#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile-destdir.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: add DESTDIR support to the -2.X Makefile.

@DPATCH@
diff -urNad mod-wsgi~/Makefile-2.X.in mod-wsgi/Makefile-2.X.in
--- mod-wsgi~/Makefile-2.X.in	2007-06-23 06:05:48.000000000 +0200
+++ mod-wsgi/Makefile-2.X.in	2007-07-30 17:33:40.000000000 +0200
@@ -13,6 +13,8 @@
 #  limitations under the License.
 
 APXS=@APXS@
+APXS_LIBEXECDIR = $(shell $(APXS) -q LIBEXECDIR)
+DESTDIR =
 PYTHON=@PYTHON@
 
 CPPFLAGS = @CPPFLAGS@
@@ -26,7 +28,7 @@
 	$(APXS) -c $(CPPFLAGS) $(CFLAGS) mod_wsgi.c $(LDFLAGS) $(LDLIBS)
 
 install : all
-	$(APXS) -i -n 'mod_wsgi' mod_wsgi.la
+	$(APXS) -i -S LIBEXECDIR=$(DESTDIR)$(APXS_LIBEXECDIR) -n 'mod_wsgi' mod_wsgi.la
 
 clean :
 	-rm -rf .libs
